### ISSUE TITLE
Docker build errors out with "ResolvePackageNotFound"

### ISSUE THREAD

**Issue by xujiboy** [ageron/handson-ml2#73]
Hi,
I am trying to use the docker build approach on my Mac as instructed [here](https://github.com/ageron/handson-ml2/tree/master/docker), but the docker build keeps erroring out at the `conda` step.
The details are shown below:
<details>
<summary>docker-compose build</summary>

    Building handson-ml2
    Step 1/18 : FROM continuumio/miniconda3:latest
    ---> 1284db959d5d
    Step 2/18 : RUN apt-get update && apt-get upgrade -y     && apt-get install -y         libpq-dev         build-essential         git         sudo         cmake zlib1g-dev libjpeg-dev xvfb ffmpeg xorg-dev libboost-all-dev libsdl2-dev swig         unzip zip     && rm -rf /var/lib/apt/lists/*
    ---> Using cache
    ---> 7eb4da419cb2
    Step 3/18 : COPY environment.yml /tmp/
    ---> Using cache
    ---> ecbe70f5bd07
    Step 4/18 : RUN conda update -y -n base conda     && echo '    - pyvirtualdisplay' >> /tmp/environment.yml     && conda env create -f /tmp/environment.yml     && conda clean -y -t     && rm /tmp/environment.yml
    ---> Running in ba433fdd00da
    Solving environment: ...working... done
    requests-2.22.0      |   90 KB | ########## | 100%
    tqdm-4.40.0          |   53 KB | ########## | 100%
    python-3.7.0         | 31.7 MB | ########## | 100%
    pycosat-0.6.3        |  105 KB | ########## | 100%
    sqlite-3.30.1        |  1.9 MB | ########## | 100%
    cryptography-2.3.1   |  585 KB | ########## | 100%
    setuptools-42.0.2    |  646 KB | ########## | 100%
    asn1crypto-1.2.0     |  162 KB | ########## | 100%
    urllib3-1.25.7       |  165 KB | ########## | 100%
    readline-7.0         |  392 KB | ########## | 100%
    tk-8.6.8             |  3.1 MB | ########## | 100%
    zlib-1.2.11          |  120 KB | ########## | 100%
    idna-2.8             |  101 KB | ########## | 100%
    _libgcc_mutex-0.1    |    3 KB | ########## | 100%
    certifi-2019.11.28   |  156 KB | ########## | 100%
    pyopenssl-19.0.0     |   82 KB | ########## | 100%
    pysocks-1.7.1        |   30 KB | ########## | 100%
    cffi-1.13.2          |  225 KB | ########## | 100%
    libedit-3.1.20181209 |  188 KB | ########## | 100%
    conda-package-handli |  872 KB | ########## | 100%
    pycparser-2.19       |   89 KB | ########## | 100%
    ca-certificates-2019 |  132 KB | ########## | 100%
    ruamel_yaml-0.15.46  |  245 KB | ########## | 100%
    six-1.13.0           |   27 KB | ########## | 100%
    openssl-1.0.2t       |  3.1 MB | ########## | 100%
    wheel-0.33.6         |   40 KB | ########## | 100%
    pip-19.3.1           |  1.9 MB | ########## | 100%
    libgcc-ng-9.1.0      |  8.1 MB | ########## | 100%
    chardet-3.0.4        |  173 KB | ########## | 100%
    conda-4.8.0          |  3.0 MB | ########## | 100%
    ## Package Plan ##

    environment location: /opt/conda

    added / updated specs:
        - conda

    The following packages will be downloaded:

        package                    |            build
        ---------------------------|-----------------
        requests-2.22.0            |           py37_1          90 KB
        tqdm-4.40.0                |             py_0          53 KB
        python-3.7.0               |       hc3d631a_0        31.7 MB
        pycosat-0.6.3              |   py37h14c3975_0         105 KB
        sqlite-3.30.1              |       h7b6447c_0         1.9 MB
        cryptography-2.3.1         |   py37hc365091_0         585 KB
        setuptools-42.0.2          |           py37_0         646 KB
        asn1crypto-1.2.0           |           py37_0         162 KB
        urllib3-1.25.7             |           py37_0         165 KB
        readline-7.0               |       h7b6447c_5         392 KB
        tk-8.6.8                   |       hbc83047_0         3.1 MB
        zlib-1.2.11                |       h7b6447c_3         120 KB
        idna-2.8                   |           py37_0         101 KB
        _libgcc_mutex-0.1          |             main           3 KB
        certifi-2019.11.28         |           py37_0         156 KB
        pyopenssl-19.0.0           |           py37_0          82 KB
        pysocks-1.7.1              |           py37_0          30 KB
        cffi-1.13.2                |   py37h2e261b9_0         225 KB
        libedit-3.1.20181209       |       hc058e9b_0         188 KB
        conda-package-handling-1.6.0|   py37h7b6447c_0         872 KB
        pycparser-2.19             |             py_0          89 KB
        ca-certificates-2019.11.27 |                0         132 KB
        ruamel_yaml-0.15.46        |   py37h14c3975_0         245 KB
        six-1.13.0                 |           py37_0          27 KB
        openssl-1.0.2t             |       h7b6447c_1         3.1 MB
        wheel-0.33.6               |           py37_0          40 KB
        pip-19.3.1                 |           py37_0         1.9 MB
        libgcc-ng-9.1.0            |       hdf63c60_0         8.1 MB
        chardet-3.0.4              |        py37_1003         173 KB
        conda-4.8.0                |           py37_0         3.0 MB
        ------------------------------------------------------------
                                            Total:        57.4 MB

    The following NEW packages will be INSTALLED:

        _libgcc_mutex:          0.1-main
        conda-package-handling: 1.6.0-py37h7b6447c_0
        tqdm:                   4.40.0-py_0

    The following packages will be UPDATED:

        asn1crypto:             0.24.0-py36_0           --> 1.2.0-py37_0
        ca-certificates:        2018.03.07-0            --> 2019.11.27-0
        certifi:                2018.4.16-py36_0        --> 2019.11.28-py37_0
        cffi:                   1.11.5-py36h9745a5d_0   --> 1.13.2-py37h2e261b9_0
        chardet:                3.0.4-py36h0f667ec_1    --> 3.0.4-py37_1003
        conda:                  4.5.4-py36_0            --> 4.8.0-py37_0
        cryptography:           2.2.2-py36h14c3975_0    --> 2.3.1-py37hc365091_0
        idna:                   2.6-py36h82fb2a8_1      --> 2.8-py37_0
        libedit:                3.1.20170329-h6b74fdf_2 --> 3.1.20181209-hc058e9b_0
        libgcc-ng:              7.2.0-hdf63c60_3        --> 9.1.0-hdf63c60_0
        openssl:                1.0.2o-h20670df_0       --> 1.0.2t-h7b6447c_1
        pip:                    10.0.1-py36_0           --> 19.3.1-py37_0
        pycosat:                0.6.3-py36h0a5515d_0    --> 0.6.3-py37h14c3975_0
        pycparser:              2.18-py36hf9f622e_1     --> 2.19-py_0
        pyopenssl:              18.0.0-py36_0           --> 19.0.0-py37_0
        pysocks:                1.6.8-py36_0            --> 1.7.1-py37_0
        python:                 3.6.5-hc3d631a_2        --> 3.7.0-hc3d631a_0
        readline:               7.0-ha6073c6_4          --> 7.0-h7b6447c_5
        requests:               2.18.4-py36he2e5f8d_1   --> 2.22.0-py37_1
        ruamel_yaml:            0.15.37-py36h14c3975_2  --> 0.15.46-py37h14c3975_0
        setuptools:             39.2.0-py36_0           --> 42.0.2-py37_0
        six:                    1.11.0-py36h372c433_1   --> 1.13.0-py37_0
        sqlite:                 3.23.1-he433501_0       --> 3.30.1-h7b6447c_0
        tk:                     8.6.7-hc745277_3        --> 8.6.8-hbc83047_0
        urllib3:                1.22-py36hbe7ace6_0     --> 1.25.7-py37_0
        wheel:                  0.31.1-py36_0           --> 0.33.6-py37_0
        zlib:                   1.2.11-ha838bed_2       --> 1.2.11-h7b6447c_3

    Downloading and Extracting Packages
    Preparing transaction: ...working... done
    Verifying transaction: ...working... done
    Executing transaction: ...working... done

    Collecting package metadata (repodata.json): ...working... done
    Solving environment: ...working... failed

    ResolvePackageNotFound:
    - tensorflow-metadata=0.15.1
    - tensorflow-addons=0.6.0

    ERROR: Service 'handson-ml2' failed to build: The command '/bin/sh -c conda update -y -n base conda     && echo '    - pyvirtualdisplay' >> /tmp/environment.yml     && conda env create -f /tmp/environment.yml     && conda clean -y -t     && rm /tmp/environment.yml' returned a non-zero code: 1
    make: *** [build] Error 1

</details>

In short, `conda` failed because of 
```
ResolvePackageNotFound:
- tensorflow-metadata=0.15.1
- tensorflow-addons=0.6.0
```
May I know if anyone has encountered similar issue and how to solve it? Many thanks~

**Comment by ageron** [comment]
Hi @xujiboy ,
Thanks for your feedback. I updated environment.yml recently, and I made a couple mistakes, sorry about that. I just fixed this, it should work on.

**Comment by xujiboy** [comment]
@ageron Thank you very much for the help~
